### PR TITLE
Record the automation status of a test case.

### PR DESCRIPTION
**Pull request by omaciel**
Make sure to record the automation status for a test case as either 'automated' or
'notautomated', based on what `Testimony` finds in the test case's docstring.

### CODE REVIEW

File: betelgeuse.py [hunk 1]
```diff
@@ -213,12 +217,15 @@ def test_case(path, collect_only, project):
                 # Ensure that a single match for the Test Case is
                 # returned.
                 assert len(results) == 1
-                test_case = results[0]
+                # Fetch the test case in order to get all of its
+                # fields and values.
+                test_case = TestCase(project, results[0].work_item_id)
```
Comment (omaciel) [inline betelgeuse.py:222]: Sadly, I have to fetch the `TestCase` object in order to access its `caseautomation` value. I tried to figure out which field to fetch from `TestCase.query` but none of the values I tried (`'caseautomation'`, `'caseautomationstatus'`) seemed to work. If we can figure out how to fetch this field from a query, then we can move this "fetch" statement back inside the conditional statement.

File: betelgeuse.py [hunk 2]
```diff
@@ -177,6 +177,10 @@ def test_case(path, collect_only, project):
                 test.docstring = test.docstring.encode(
                     'ascii', 'xmlcharrefreplace')
 
+            # Is the test automated? Acceptable values are:
+            # automated, manualonly, and notautomated
+            auto_status = 'notautomated' if test.automated else 'automated'
```
Comment (omaciel) [inline betelgeuse.py]: Since `test.automated` is reported by `Testimony`, we either get a `True` or `False`.

Comment (sthirugn) [inline betelgeuse.py]: Shouldn't 'automated' and 'notautomated' be reversed here in this statement?

**Comment by sthirugn** [comment]
Pending question

File: betelgeuse.py (hunk 2 above)
Comment (omaciel) [inline betelgeuse.py]: I must have left my brain out in the snow over the weekend :/ I'll fix it

Comment (sthirugn) [inline betelgeuse.py]: no worries

**Comment by sthirugn** [comment]
ACK

**Comment by omaciel** [comment]
Quick update: I checked with the `Pylarion` lead developer and he recommended that I do exactly what my changes introduced, i.e., fetch the `TestCase` to get access to all of its fields and values. The recommended alternative did not work (since `caseautomation` is a custom field, try to query for the `_custom_fields` field).

``` python
Sadly that didn't work :/

>>> foo = TestCase.query('CLOUDTP-11279', fields=['description', 'work_item_id', '_custom_fields'])
>>> bar = foo[0]
>>> bar._custom_fields
[]
```

**Comment by elyezer** [comment]
ACK

**Pull request merged**